api/turing/imagebuilder/imagebuilder.go

```diff
@@ -277,6 +276,34 @@ func (ib *imageBuilder) createKanikoJob(
 		annotations["cluster-autoscaler.kubernetes.io/safe-to-evict"] = "false"
 	}
 
+	var volumes []cluster.SecretVolume
+	var volumeMounts []cluster.VolumeMount
+	var envVars []cluster.Env
+
+	// If kaniko service account is not set, use kaniko secret
+	if ib.imageBuildingConfig.KanikoConfig.ServiceAccount == "" {
+		kanikoArgs = append(kanikoArgs,
```

Comment:
This is already set when `kanikoArgs` is initialised, that line should be removed since it's now being set conditionally.

Reply:
Ah oh yes, nice catch 😅 Just removed it! :)